source/main.c

```diff
@@ -38,6 +37,18 @@ struct shortcut {
 };
 int num_shortcuts = sizeof(shortcuts)/sizeof(shortcuts[0]);
 
+u16 all_buttons_held;
+void scan_all_buttons_held()
```

Comment:
I realize this is named after the function it calls, but this name feels a bit awkward.